napari/utils/config.py

```diff
@@ -21,27 +22,11 @@ def _set(env_var: str) -> bool:
 
 Async Loading
 -------------
-Image layers will use the ChunkLoader to load data instead of loading
-the data directly. Image layers will not call np.asarray() in the GUI
-thread. The ChunkLoader will call np.asarray() in a worker thread. That
-means any IO or computation done as part of the load will not block the
-GUI thread.
-
-Set NAPARI_ASYNC=1 to turn on async loading with default settings.
+Deprecated.
 
 Octree Rendering
 ----------------
-Image layers use an octree for rendering. The octree organizes the image
-into chunks/tiles. Only a subset of those chunks/tiles are loaded and
-drawn at a time. Octree rendering is a work in progress.
-
-Enabled one of two ways:
-
-1) Set NAPARI_OCTREE=1 to enabled octree rendering with defaults.
-
-2) Set NAPARI_OCTREE=/tmp/config.json use a config file.
-
-See napari/utils/_octree.py for the config file format.
+Deprecated.
```

Comment:
Similar comment as for https://github.com/kcpevey/napari/pull/28/files#r1162150144